packages/next/pages/_document.tsx

```diff
@@ -71,10 +72,73 @@ function getPolyfillScripts(context: HtmlProps, props: OriginProps) {
     ))
 }
 
+function getPreNextWorkerScripts(context: HtmlProps, props: OriginProps) {
+  const { scriptLoader, crossOrigin, optimizeScripts } = context
+
+  if (!optimizeScripts) return null
+
+  try {
+    let {
+      partytownSnippet,
+    } = require(/* webpackIgnore: true */ '@builder.io/partytown/integration'!)
+
+    const children = Array.isArray(props.children)
+      ? props.children
+      : [props.children]
+
+    // Check to see if the user has defined their own Partytown configuration
+    const userDefinedConfig = children.find(
+      (child: React.ReactElement) =>
+        child?.props?.dangerouslySetInnerHTML?.__html.length &&
+        'data-partytown-config' in child.props
+    )
+
+    return (
+      <>
+        {!userDefinedConfig && (
+          <script
+            data-partytown-config=""
+            dangerouslySetInnerHTML={{
+              __html: `
+            partytown = {
+              lib: "/_next/static/~partytown/"
```

Comment:
```suggestion
              lib: "${assetPrefix}/_next/static/~partytown/"
```
Do we need to apply the `assetPrefix` here?

Reply:
Ah good catch. Added (and also included a note to the docs if anyone ends up overwriting the config).